node/builder_chain.go

```diff
@@ -236,6 +239,15 @@ func ConfigFullNode(c interface{}) Option {
 			Unset(new(*wallet.LocalWallet)),
 			Override(new(wallet.Default), wallet.NilDefault),
 		),
+		// Chain node cluster enabled
+		If(cfg.Raft.ClusterModeEnabled,
+			Override(new(*gorpc.Client), modules.NewRPCClient),
+			Override(new(*consensus.Consensus), consensus2.NewConsensusWithRPCClient(false)),
+			Override(new(*messagesigner.MessageSignerConsensus), messagesigner.NewMessageSignerConsensus),
+			Override(new(messagesigner.MsgSigner), func(ms *messagesigner.MessageSignerConsensus) *messagesigner.MessageSignerConsensus { return ms }),
```

Comment:
Can use the `From` helper

```suggestion
			Override(new(messagesigner.MsgSigner), From(new(*messagesigner.MessageSignerConsensus))),
```